test/browser-only/__snapshots__/constructLayoutEngine-browser.test.js.snap

```diff
@@ -392,17 +392,27 @@ exports[`constructLayoutEngine browser error handling works with error handlers
 <body>
   <div
     id="single-spa-application:header"
-  />
+  >
+    <div>
```

Comment:
This changed because I modified the test to fire the error handler for the header.